frontend/src/routes/(authed)/api/recent-match/+server.ts

```diff
@@ -0,0 +1,20 @@
+import type { RequestHandler } from './$types';
+
+export const GET: RequestHandler = async ({ url, locals: { apiClient } }) => {
+  const playerId = Number(url.searchParams.get('playerId'));
+  if (Number.isNaN(playerId)) {
+    throw new Error('Invalid player ID');
+  }
+  const latestMatch = await apiClient.mmrApi.v2MmrMatchesGet({
+    userId: playerId,
+    limit: 1,
+    offset: 0,
+  });
+
+  return new Response(
+    JSON.stringify({
+      playerId,
+      latestMatch: latestMatch[0],
+    })
+  );
+};
```

Comment:
You can delete all of this if you can use the apiClient directly on the client

Reply:
See other comment. Even if I technically could due to supabase I don't want us to build upon it, since we'll eventually move away from this.